flare/otf.py

```diff
@@ -514,6 +574,18 @@ def record_state(self):
             self.velocities,
         )
 
+    def record_dft_data(self, structure, target_atoms):
+        # write xyz and energy/forces/stress
+        self.output.write_xyz_config(
+            self.curr_step,
+            self.structure,
+            self.structure.forces,
+            self.structure.stds,
+            dft_forces=self.structure.forces,
+            dft_energy=self.structure.potential_energy,
+            target_atoms=target_atoms,
+        )
+
```

Comment:
Should be helpful for keeping track of the sparse environments. I would include this in the PR description too

Reply:
added to the description